src/videoCache.js

```diff
@@ -59,13 +59,20 @@ function wrapURI(uri, impUrl) {
  * @param {CacheableBid} bid
  */
 function toStorageRequest(bid) {
+  const conf = config.getConfig();
   const vastValue = bid.vastXml ? bid.vastXml : wrapURI(bid.vastUrl, bid.vastImpUrl);
+
   let payload = {
     type: 'xml',
     value: vastValue,
     ttlseconds: Number(bid.ttl)
   };
 
+  if (conf.cache && conf.cache.url && conf.cache.vasttrack) {
```

Comment:
Don't think this will ever be hit if `conf.cache.url` is not defined, the check is made in https://github.com/prebid/Prebid.js/blob/master/src/auction.js#L414 which then calls `store`

 
Can probably just do:
```javascript
if (config.getConfig('cache.vasttrack') {
```

But doesn't hurt too much to have those safe checks I guess.

Reply:
Yeah, it was a bit of a defensive approach. I like cleaning up the code. Pushing update.